velox/dwio/dwrf/common/Decryption.cpp

```diff
@@ -59,7 +59,9 @@ void DecryptionHandler::populateChildNodeMap(
 std::unique_ptr<DecryptionHandler> DecryptionHandler::create(
     const Footer& footer,
     DecrypterFactory* factory) {
-  return create(*footer.getDwrfFooter(), factory);
+  return footer.format() == DwrfFormat::kDwrf
```

Comment:
This is to avoid changing upper layer code.
`DecryptionHandler::create` is called in `ReaderBase` for both formats, and its `handler_` is expected to be non-null.